staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -1813,7 +1813,7 @@ type CSIPersistentVolumeSource struct {
 
 // Represents a source location of a volume to mount, managed by an external CSI driver
 type CSIVolumeSource struct {
-	// Driver is the name of the CSI driver that handles this volume.
+	// driver is the name of the CSI driver that handles this volume.
 	// Consult with your admin for the correct name as registered in the cluster.
 	Driver string `json:"driver" protobuf:"bytes,1,opt,name=driver"`
 
```

Comment:
same comment about `readOnly` and `fsType` fields below

Reply:
addressed this too. ptal. thanks..